sandboxes/staticanalysis/Dockerfile

```diff
@@ -1,10 +1,23 @@
 FROM golang:1.19-alpine@sha256:122f3484f844467ebe0674cf57272e61981770eb0bc7d316d1f0be281a88229f as build
 
-WORKDIR /src
-COPY . ./
+# Note: Dockerfile uses paths relative to the top-level project directory,
+# so it should be built from that directory, i.e:
+# $ cd package-analysis
+# $ docker build -f sandboxes/staticanalysis/Dockerfile .
 
+# Cache dependencies to avoid downloading again on code change
 WORKDIR /src/sandboxes/staticanalysis
+# Dependencies for package analysis
+COPY ./go.mod /src/
```

Comment:
nit: you can have both on the same COPY line

i.e.

```
COPY go.mod go.sum /src/
```

Same for below.

Reply:
Oh great! Thanks for the suggestion :D